src/bridge/MemberListSyncer.ts

```diff
@@ -264,6 +264,11 @@ export class MemberListSyncer {
         const idleRegex = this.server.ignoreIdleUsersOnStartupExcludeRegex;
         for (const roomInfo of filteredRooms) {
             for (const uid of roomInfo.realJoinedUsers) {
+                const banReason = this.ircBridge.matrixBanSyncer?.isUserBanned(uid);
+                if (banReason) {
+                    log.debug(`Not syncing ${uid} - user banned (${banReason})`)
```

Comment:
I this just to avoid syncing the presence of the user in the room to the IRC side?

Reply:
Yep, it's just a shortcut to avoid having to do any processing on users we know will fail to get connected later.